discojs/src/aggregator/mean.ts

```diff
@@ -122,6 +128,7 @@ export class MeanAggregator extends Aggregator<WeightsContainer> {
     this.log(AggregationStep.AGGREGATE);
 
     const result = aggregation.avg(currentContributions.values());
+    this._round = this._round++
```

Comment:
this is a double operation which is confusing

```suggestion
    this._round++
```